lib/web/apiserver.go

```diff
@@ -77,8 +77,8 @@ import (
 )
 
 const (
-	// ssoLoginConsoleErr is a generic error message to hide revealing sso login failure msgs.
-	ssoLoginConsoleErr = "Failed to login. Please check Teleport's log for more details."
+	// SSOLoginConsoleErr is a generic error message to hide revealing sso login failure msgs.
```

Comment:
```suggestion
	// SSOLoginConsoleErr is a generic error message avoid disclosing sensitive SSO failure messages.
```

When something is exported, it's extra important to get the name and documentation right.

Additionally, I would generally expect that something named `FooErr` implements `error`, and this is just a string. Maybe `SSOLoginFailureMessage` or something along those lines would be a better name.

Reply:
ack. renamed and updated doc string as suggested.